quay.go

```diff
@@ -22,38 +23,49 @@ type QuayTagsResponse struct {
 	Tags          []QuayTag `json:"tags"`
 }
 
-func constructQuayURL(image string) (string, error) {
+func constructQuayURL(image string, Page int) (string, error) {
```

Comment:
A private variable should be lowercase.

```suggestion
func constructQuayURL(image string, page int) (string, error) {
```